GG/GG/ZList.h

```diff
@@ -59,14 +59,14 @@ class GG_API ZList
     /** RenderOrderIterable can be iterated in back to front render order. */
     struct RenderOrderIterable {
         RenderOrderIterable(const std::list<std::shared_ptr<Wnd>>& list) :
-            m_list(list.crbegin(), list.crend())
+            m_list(list.rbegin(), list.rend())
```

Comment:
seems unrelated to the subject of this pull request...

Reply:
See #2730